File: src/main/java/com/muxiu1997/sharewhereiam/loader/IntegrationLoader.java
package com.muxiu1997.sharewhereiam.loader;

import com.muxiu1997.sharewhereiam.integration.Mods;

import java.util.Arrays;

public class IntegrationLoader {

    public static final IntegrationLoader INSTANCE = new IntegrationLoader();

    public void load() {
        Arrays.stream(Mods.values()).forEach(Mods::check);
    }
}

File: src/main/kotlin/com/muxiu1997/sharewhereiam/loader/IntegrationLoader.kt
package com.muxiu1997.sharewhereiam.loader

import com.muxiu1997.sharewhereiam.integration.Mods
import java.util.*

object IntegrationLoader {
    fun load() {
        Arrays.stream(Mods.values()).forEach { obj: Mods -> obj.check() }
    }
}
